M/MolecularGraph/Deps.toml

```diff
@@ -11,3 +11,6 @@ InteractiveUtils = "b77e0a4c-d291-57a0-90e8-8db25a27a240"
 Pkg = "44cfe95a-1eb2-52ea-b672-e2afdf69b78f"
 Profile = "9abbd945-dff8-562f-b5e8-e1ebf5ef1b79"
 YAML = "ddb6d928-2868-570f-bddf-ab3f9cf99eb6"
+
+["0.3.1-0"]
+Debugger = "31a5f54b-26ea-5ae9-a837-f05ce5417438"
```

Comment:
Did you intend this? Doesn't seem to be used.

Reply:
No. This is just a development tool.
I have to remove it and some other development tools. Please let me revise the file.
Thank you for your suggestion.